src/plugins/irc-events/message.js

```diff
@@ -109,16 +109,14 @@ module.exports = function(irc, network) {
 			let title = data.nick;
 
 			if (chan.type !== Chan.Type.QUERY) {
-				title += ` (${chan.name}) mentioned you`;
-			} else {
-				title += " sent you a message";
+				title += ` (${chan.name})`;
```

Comment:
Any chance you can/want to use the same wording than for desktop notifications? "says:", etc.

Reply:
The shorter it is, the better, imo. The way it groups messages looks a little messy, so we might something like bullet points.

Reply:
Alright then.